src/main/java/frc/robot/RobotContainer.java

```diff
@@ -104,35 +96,18 @@ public RobotContainer() {
    */
   private void configureButtonBindings() {
     outtakeButton
-        .whenActive(new RunOuttakeCommand(outtake, Constants.OuttakeConstants.OUTTAKE_SPEED))
-        .whenInactive(new StopOuttakeCommand(outtake));
+        .whileHeld(new RunOuttakeCommand(outtake, Constants.OuttakeConstants.OUTTAKE_SPEED));
     towerButton
-        .whenActive(new RunTowerCommand(tower, TowerConstants.TOWER_SPEED))
-        .whenInactive(new StopTowerCommand(tower));
+        .whileHeld(new RunTowerCommand(tower, TowerConstants.TOWER_SPEED));
     climberUpButton
-        .whenActive(new RunClimberCommand(climber, ClimberConstants.CLIMBER_SPEED))
-        .whenInactive(new StopClimberCommand(climber));
+        .whileHeld(
+            new RunClimberCommand(leftClimber, rightClimber, ClimberConstants.CLIMBER_SPEED));
     climberDownButton
-        .whenActive(new RunClimberCommand(climber, -ClimberConstants.CLIMBER_SPEED))
-        .whenInactive(new StopClimberCommand(climber));
-
-    configureIntakeButton(frontIntake, frontIntakeButton);
-    configureIntakeButton(backIntake, backIntakeButton);
-  }
-
-  /**
-   * Configures a button for the intake. Extends intake and then spins intake while a button is
-   * held. Stops intake and then retracts intake when the button is released.
-   * 
-   * @param intake The intake to control
-   * @param button The button to configure
-   */
-  private void configureIntakeButton(Intake intake, Button button) {
-    button
-        .whenActive(new SequentialCommandGroup(new ExtendIntakeCommand(intake),
-            new SpinIntakeCommand(intake, IntakeConstants.INTAKE_SPEED)))
-        .whenInactive(new SequentialCommandGroup(new StopSpinIntakeCommand(intake),
-            new RetractIntakeCommand(intake)));
+        .whileHeld(
+            new RunClimberCommand(leftClimber, rightClimber, -ClimberConstants.CLIMBER_SPEED));
+    intakeButton
+        .whileHeld(new SequentialCommandGroup(new ExtendIntakeCommand(intake),
```

Comment:
I don’t think the code ever retracts the intake.

Reply:
It sets the default command to retract

Reply:
Like with tower/outtake/climber

Reply:
I see. But there is no stop command during teleop. Maybe you should add the stop commands to the teleop default command.

Reply:
Yeah I asked in the slack abt why default commands are set during auton and not teleop
Might be a bug but I'll wait for a response before changing anything